riotfile.py

```diff
@@ -88,7 +88,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
     venvs=[
         Venv(
             pys=["3"],
-            pkgs={"black": "==21.4b2", "isort": [latest]},
+            pkgs={"black": "==21.4b2", "isort": [latest], "click": "<8.1.0"},
```

Comment:
Can we add a comment referencing https://github.com/psf/black/issues/2964 so it can be removed in the future when the issue is addressed?